project/plugin-scalajs.sbt

```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.26")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.27")
```

Comment:
I'm assuming this is a safe upgrade, but I know nothing about scalajs.  Availability forced my hand here.

Reply:
If the tests all pass, it should be fine. ScalaJS support is still unofficial.

Reply:
I'm not yet sure what's happening [here](https://travis-ci.org/Log4s/log4s/jobs/518565691#L918).  Everything passes for me locally.  Is this a problem you're familiar with?  If not, I'll dig deeper.

Reply:
I tried tag 1.7.0 with oraclejdk8 to see if the Travis problem predates 0.6.27, and ran into a [different flicker](https://travis-ci.org/rossabaker/log4s/jobs/519534281#L925).

All of the problems are scala.js, where I'm a bit lost.

Reply:
Could you try adding these settings (probably one at a time) to `build.sbt`:
`parallelExecution in (ThisBuild, Test) := false`

`concurrentRestrictions in Global += Tags.limit(ScalaJSTags.Link, 2)`

Reply:
I'm not seeing this on the recent builds.  I don't know what would have fixed it, so it may be intermittent, but right now the only failure is [this old one](https://github.com/Log4s/log4s/pull/32#issuecomment-487329648).

Reply:
Looks like some state between tests is being preserved in the js side. maybe it is a scalatest issue?

Reply:
Unless I'm reading the stack traces wrong (which I did earlier!), it's not just a JS issue.  It's also failing on the JVM in the same way.